docs/IE_DG/Int8Inference.md

```diff
@@ -111,17 +63,27 @@ available from the Inference Engine API. The layers have the following marks:
 * Suffix `I8` for layers that had 8-bit data type input and were computed in 8-bit precision
```

Comment:
I think we need to put this information right after "The execType column of the table includes inference primitives with specific suffixes."